client/components/swipeable/index.js

```diff
@@ -0,0 +1,212 @@
+import classnames from 'classnames';
+import { Children, useState, useEffect, useRef } from 'react';
+
+import './style.scss';
+
+export const Swipable = ( {
+	hasDynamicHeight = false,
+	children,
+	currentPage = 0,
+	onPageSelect,
+	pageClassName,
```

Comment:
I think we can always add this later but I don't think it's needed at this point.

Reply:
It can be added later — I just think it's good practice for a component to forwards its props, and I thought it would be a minor change that could be added to this PR.

Reply:
Sorry I didn't realize that it was a good practice. Added.